extensions/rvadapter/src/main/java/com/google/android/agera/rvadapter/RepositoryPresenterCompiler.java

```diff
@@ -137,6 +150,22 @@ public RPMain recycleWith(@NonNull final Receiver recycler) {
     return this;
   }
 
+  @NonNull
+  @Override
+  public RPMain diffWith(@NonNull final Function keyForItem, final boolean detectMoves) {
+    this.keyForItem = keyForItem;
+    this.detectMoves = detectMoves;
+    return this;
+  }
+
+  @NonNull
+  @Override
+  public RPItemCompile diff() {
+    this.keyForItem = SAME_KEY_FOR_ITEM;
+    this.detectMoves = false;
```

Comment:
I'm guessing this is because you can't move a single item, only add/remove/change?